noaa_semantic_search/workload.json

```diff
@@ -0,0 +1,32 @@
+{% import "benchmark.helpers" as benchmark with context %}
+
+{
+  "version": 2,
+  "description": "Global daily weather measurements from NOAA",
```

Comment:
lets include some reference to what this workload will execute

Reply:
ack